.gitignore

```diff
@@ -87,4 +85,4 @@ dev/src/dev/nocommit/
 .calva/
 
 # malli + clj-kondo: ignore malli-types
-.clj-kondo/metosin/malli-types-clj
+.clj-kondo/metosin/malli-types
```

Comment:
This showed up after I merged master — not sure how. What should I do to resolve this?

Reply:
`git rebase master`